dnplab/dnpIO/bes3t.py

```diff
@@ -0,0 +1,237 @@
+import numpy as np
+import os
+from dnplab import dnpdata
+import warnings
+
+
+def import_bes3t(path):
+    """
+    Import Bruker BES3T data and return dnpdata object
+
+    Args:
+        path (str) : Path to either .DSC or .DTA file
+
+    Returns:
+        bes3t_data (object) : dnpdata object containing Bruker BES3T data
+    """
+
+    pathexten = path[-3:]
+    path = path[:-3]
```

Comment:
Here I would use os.path.splitext to split the path and extension.

Reply:
i made the change. once all comments are made and addressed I will make a final commit before the merge